### ISSUE TITLE
Text size, zooming etc.

### ISSUE THREAD

**Issue by Ravenbird** [beakerbrowser/beaker#842]
Operation System: Ubuntu Linux 16.04
Beaker Version: 0.7.11
Package: AppImage

I can't adjust the size of the text in the Beaker browser or zoom the page. The menus of the Beaker browser are also quite small compared to the rest of the system. The latter may also be due to the fact that it is installed via AppImage.

**Comment by pfrazee** [comment]
Could you show a screenshot? I'm wondering if this is a bug

**Comment by Ravenbird** [comment]
This is how it looks for me. On the left side the beaker browser and on the right side the firefox browser.

![beaker-screenshot](https://user-images.githubusercontent.com/2757648/35930362-fd8a3a8e-0c31-11e8-830e-7511563c8326.jpg)

**Comment by taravancil** [comment]
It looks like your Firefox is zoomed in to 120% for GitHub. Are you able to control the zoom in Beaker with View -> Zoom In?

**Comment by pfrazee** [comment]
Try hitting the `alt` key to get the window menu.

**Comment by Ravenbird** [comment]
Ah, the keyboard shortcuts are slightly different than I've known them before. In fact, I have turned the zoom level of my Firefox up a bit, as I have the screen a little more than normal away from me. The text of the graphical user interface (XFCE) is also slightly enlarged. Apparently AppImage does not adapt to this. But as I got it right now with the zoom, it fits for the time being. Thanks for your help! :-)
